ext/stringio/stringio.c

```diff
@@ -1366,11 +1368,13 @@ strio_getline(struct getline_arg *arg, struct StringIO *ptr)
 
 /*
  * call-seq:
- *   strio.gets(sep=$/, chomp: false)     -> string or nil
- *   strio.gets(limit, chomp: false)      -> string or nil
- *   strio.gets(sep, limit, chomp: false) -> string or nil
+ *   gets(sep = $/, **line_opts) → string or nil
```

Comment:
Minor: Here, and in the rest of the change, the arrow is a ligature.

I don't know which is preferred/correct.

Reply:
Fixed.